inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/StreamSinkEntityMapper.java

```diff
@@ -161,6 +161,6 @@ List<String> selectExistsStreamId(@Param("groupId") String groupId, @Param("sink
      */
     int deleteByInlongGroupIds(@Param("groupIdList") List<String> groupIdList);
 
-    String selectAssignedCluster(@Param("dataNodeName") String dataNodeName);
+    String selectAssignedCluster(@Param("dataNodeName") String dataNodeName, @Param("clusterTag") String clusterTag);
```

Comment:
Does this modification only affect SortStandalone?

Reply:
yes, only StandaloneResourceOperator call this method